fw/auton_led/auton_led.ino

```diff
@@ -0,0 +1,47 @@
+int red_led = 12;
+int green_led = 10;
+int blue_led = 11;
+
+void led_control(char led_ID) {
+    if (led_ID == 'r') {
+        digitalWrite(red_led, LOW);
+        digitalWrite(green_led, HIGH);
+        digitalWrite(blue_led, HIGH);
+
+    } else if (led_ID == 'g') {
+        digitalWrite(green_led, LOW);
+        digitalWrite(blue_led, HIGH);
+        digitalWrite(red_led, HIGH);
+    }
+
+    else if (led_ID == 'b') {
+        digitalWrite(blue_led, LOW);
+        digitalWrite(red_led, HIGH);
+        digitalWrite(green_led, HIGH);
+    }
+
+    else if (led_ID == 'o') {
+        digitalWrite(blue_led, HIGH);
```

Comment:
why is HIGH off and LOW on, just curious?

Reply:
The circuit is as follows: 5V to GPIO. If 5V to High, then there is no current diff and the LED doesn't shine. If 5V to low, then there is a voltage drop and current goes and the LED shines.